archimedes/archimedes/runner.py

```diff
@@ -181,6 +196,13 @@ def reserve_exec_dir(self) -> ContextManager[str]:
 
         return tempfile.TemporaryDirectory(dir=self.exec_dir)
 
+    def _cmd(self, request: RunRequest, exec_script_path: str):
+        if request.run_with("node"):
+            exec_dir = exec_script_path.replace(request.target_file, "")
+            return ["/bin/bash", "-c", f"cd {exec_dir}; cp /app/package.json package.json; npm install; node {exec_script_path}"]
```

Comment:
Copying over the `package.json` and re-installing node modules in the same directory as the injected Vulcan script, because `node` expects to find them available locally and not globally. Hoping this isn't a burden because the dependencies should have been installed already and in the `npm` cache.